### ISSUE TITLE
Custom buttons no longer show toggled state

### ISSUE THREAD

**Issue by brunnerh** [nhn/tui.editor#1592]
## Describe the bug
In v2 toolbar buttons could e.g. use a `state` like `bold` to create a button that behaves the same as the default bold button.
If the selected text in the document is then bold, the button would reflect that.

In v3 this seems to no longer be supported.

## To Reproduce
1. Add a custom toolbar button along these lines:
  ```js
  const button = document.createElement('button');
  button.type = 'button';
  button.addEventListener('click', () => editor.exec('bold'));
  
  const item = {
  	el: button,
  	state: 'bold', // No longer seems to do anything.
  	command: 'bold', // Also does not matter? Had to attach a click handler manually.
  };
  ```
2. Select text and use button
3. The button does not get a class applied, that indicates that the selection is already bold.

## Expected behavior
A class is applied to signal that a given command's state is active.

## Desktop (please complete the following information):
 - OS: Windows 10
 - Browser Chrome
 - Version: 91

**Comment by js87zz** [comment]
@brunnerh 
If you want to use our basic button UI, you can do as below. (The `active` class will added to button class.)
```js
const item = {
  name: 'myItem'
  state: 'strong', // The existing bold state should be changed to strong in 3.0
  command: 'bold',
  text: // ...,
  className: 'toastui-editor-toolbar-icons first',
  style: { backgroundImage: 'none' }
};
```
However, I checked that it was not worked to activate state with the customizing element.
I will fix it. Thanks!

**Comment by brunnerh** [comment]
(`state` is also no longer in the type definitions for `ToolbarItemOptions`)

**Comment by js87zz** [comment]
@brunnerh 
This feature has been fixed in `v3.0.1`
You can see the tutorial [here](https://github.com/nhn/tui.editor/blob/master/docs/en/toolbar.md#change-toolbar-item-state)
